util/chunk/chunk.go

```diff
@@ -133,6 +148,25 @@ func newVarLenColumn(cap int, old *column) *column {
 	}
 }
 
+// RequiredRows returns how many rows is considered full.
+func (c *Chunk) RequiredRows() int {
+	return c.requiredRows
+}
+
+// SetRequiredRows sets the number of required rows.
+func (c *Chunk) SetRequiredRows(requiredRows int) *Chunk {
```

Comment:
can we change this function signature to `SetRequiredRows(requiredRows maxChunkSize int)`? So that we don't need to hold a `maxChunkSize` copy in `Chunk`? I found this is the only place to use this variable?

Reply:
Ok, `maxChunkSize` has been removed from `Chunk`.